src/input-helper.ts

```diff
@@ -3,17 +3,20 @@ import * as core from '@actions/core';
 export interface ActionInputs {
   workingDirectory: string;
   commentStrategy: 'always' | 'skip-insignificant';
+  createIssue: boolean;
   githubToken: string;
 }
 
 export function getInputs(): ActionInputs {
   const workingDirectory = core.getInput('working-directory');
   const commentStrategy = core.getInput('comment-strategy');
+  const createIssue = core.getInput('create-issue');
   const githubToken = core.getInput('github-token');
 
   const inputs = {
     workingDirectory,
     commentStrategy: commentStrategy === 'skip-insignificant' ? 'skip-insignificant' : 'always',
+    createIssue: createIssue !== 'no' && createIssue !== 'false',
```

Comment:
Can we limit this to just boolean values and document the supported values to be `'true'` and `'false'` in the README and action.yml?
```suggestion
    createIssue: createIssue !== 'false',
```